### ISSUE TITLE
[td-tools] - 'webpack' is not recognized as an internal or external command

### ISSUE THREAD

**Issue by danielpeintner** [eclipse-thingweb/node-wot#1257]
Recently I noticed the following issue when running
`npm run build` in the `/packages/td-tools/` folder 

```
> tsc -b && webpack

'webpack' is not recognized as an internal or external command,
operable program or batch file.
```

Running `npm run build` in the root folder does not cause any issue. Is it a local issue on my machine (which I doubt) or is there anything that has been changed I am not aware of!?

**Comment by relu91** [comment]
> Running npm run build in the root folder does not cause any issue. Is it a local issue on my machine (which I doubt) or is there anything that has been changed I am not aware of!?

Correct, this is due to we now building the whole project using tsc and we are not iterating on the packages list like before (i.e., we are not calling npm run build inside the `td-tools` folder anymore). this is another reason why we have to call `npm run build -w packages/browser-bundle` manually in the root `build` script. Anyhow, as mentioned in https://github.com/eclipse-thingweb/node-wot/issues/387#issuecomment-2015152086 I hope to migrate soon to esbuild then `webpack` would not be needed anymore in td-tools.

**Comment by relu91** [comment]
Sorry I might be out of sync with the work done with the td-tools, but if I recall correctly, we should eventually move everything to the new repository. This means that this issue is no relevant anymore or should be move to https://github.com/eclipse-thingweb/td-tools

**Comment by relu91** [comment]
should be closed after https://github.com/eclipse-thingweb/node-wot/issues/1301

**Comment by danielpeintner** [comment]
Yes, for now the issue is still present.
